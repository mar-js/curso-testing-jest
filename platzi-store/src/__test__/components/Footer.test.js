import React from 'react';
import { create } from 'react-test-renderer';
import { mount } from 'enzyme';

import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const FOOTER = mount(<Footer />);

  test('Render component <Footer />', () => expect(FOOTER.length).toEqual(1));

  test('Render title <Footer />', () => expect(FOOTER.find('.Footer-title').text()).toEqual('Platzi Store'));
});

describe('<Footer /> Snapshot', () => test('UI component <Footer />', () => {
  const FOOTER = create(<Footer />);
  expect(FOOTER.toJSON()).toMatchSnapshot();
}));
