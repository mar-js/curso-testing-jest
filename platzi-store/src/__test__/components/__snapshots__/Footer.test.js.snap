// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Snapshot UI component <Footer /> 1`] = `
<div
  className="Footer aaaaaaaaaaaaaa"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los ... Reservados
  </p>
</div>
`;
